app/retail/templates/shared/visionhero.html

```diff
@@ -0,0 +1,39 @@
+{% comment %}
+    Copyright (C) 2017 Gitcoin Core
+
```

Comment:
can we pls not make another hero for this and instead use the common hero? i'm okay if it's not exactly whats in the design if it means we dont repeat ourselves in the code

Reply:
sorry I forgot to exclude this one, I was just testing Didn't used I will remove it